package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hashlru",
   "description": "simpler faster substitute for LRU",
-  "version": "2.2.1",
+  "version": "2.3.1",
```

Comment:
please leave the maintainer to set the version number when they merge, don't set it in the PR.